ci-operator/config/openshift/kueue-operator/openshift-kueue-operator-release-1.0.yaml

```diff
@@ -178,6 +192,63 @@ tests:
           memory: 500Mi
     test:
     - chain: kueue-operator-test-e2e
+- always_run: false
```

Comment:
Given the goal is to have a periodic disconnect job, I believe this can be remove as always_run is meant for presubmit

Reply:
Sure, let me update that.